src/classes/client.js

```diff
@@ -243,7 +243,7 @@ class Komada extends Discord.Client {
    */
   get invite() {
     if (!this.user.bot) throw "Why would you need an invite link for a selfbot...";
-    const permissions = Discord.Permissions.resolve([...new Set(this.commands.reduce((a, b) => a.concat(b.conf.botPerms), ["READ_MESSAGES", "SEND_MESSAGES"]))]);
+    const permissions = Discord.Permissions.resolve([...new Set(this.commands.reduce((a, b) => a.concat(b.conf.botPerms), ["VIEW_CHANNEL", "SEND_MESSAGES"]))]);
```

Comment:
Consider ``Discord.Permissions.resolve([...(new Set(mySet))]``. You're using a spread operator on a `new` keyword.